web/thesauruses/_meta/lists.json

```diff
@@ -59,13 +49,6 @@
       "access_element_in_ordered_immutable_list": "Access element by index",
       "delete_ordered_immutable_list": "Delete the list"
     },
-    "Sorted Immutable Lists": {
-      "name_of_sorted_immutable_list": "What is a sorted immutable list called?",
-      "create_a_sorted_immutable_list": "Create the list",
-      "sorted_immutable_list_start_number": "What number does it start at?",
-      "access_element_in_sorted_immutable_list": "Access element by index",
-      "delete_sorted_immutable_list": "Delete the list"
-    },
```

Comment:
A list that keeps elements sorted on insertion doesn't make any sense if we don't have any insertions.

Reply:
And thirdly makes sense too.